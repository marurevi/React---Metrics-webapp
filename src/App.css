@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap');

*,
*::before,
*::after {
  box-sizing: border-box;
  font-family: sans-serif;
  text-align: center;
  font-family: 'Lato', sans-serif;
}

body {
  background-color: #000;
  color: rgb(236,76,138);
}
