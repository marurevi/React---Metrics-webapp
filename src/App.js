import './App.css';
import { Routes, Route } from 'react-router-dom';

function App() {
  return (
    <div className="App">
      <Navigator />
      <Routes>
        <Route index element={Home} />
        <Route path="/details" element={Details} />
      </Routes>
      CAPSTONE
    </div>
  );
}

export default App;
