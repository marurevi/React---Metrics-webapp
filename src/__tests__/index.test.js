import { Provider } from 'react-redux';
import { PersistGate } from 'redux-persist/lib/integration/react';
import { BrowserRouter as Router } from 'react-router-dom';
import { render, act } from '@testing-library/react';
import App from '../App';
import store, { persistor } from '../redux/configureStore';

test('Should match the snapshot', async () => {
  const dom = render(
    <Provider store={store}>
      <PersistGate loading={null} persistor={persistor}>
        <Router>
          <App />
        </Router>
      </PersistGate>
    </Provider>,
  );
  await act(() => expect(dom).toMatchSnapshot());
});
