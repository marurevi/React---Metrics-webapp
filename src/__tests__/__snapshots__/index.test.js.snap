// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match the snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div>
          <nav
            class="sc-bczRLJ gtcTJi"
          >
            <svg
              class="sc-dkzDqf jhxCHO"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
              />
            </svg>
            <div
              class="sc-jSMfEi dBlUfo"
            >
              <a
                aria-current="page"
                class="sc-gsnTZi eWKMPY active"
                href="/"
              >
                Home
              </a>
              <a
                activestyle="true"
                class="sc-gsnTZi eWKMPY"
                href="/country"
              >
                Country
              </a>
            </div>
          </nav>
        </div>
        <div>
          <h1>
            Covid-19 Metrics
          </h1>
          <div
            class="sc-ftvSup kEhnWs"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div>
        <nav
          class="sc-bczRLJ gtcTJi"
        >
          <svg
            class="sc-dkzDqf jhxCHO"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
            />
          </svg>
          <div
            class="sc-jSMfEi dBlUfo"
          >
            <a
              aria-current="page"
              class="sc-gsnTZi eWKMPY active"
              href="/"
            >
              Home
            </a>
            <a
              activestyle="true"
              class="sc-gsnTZi eWKMPY"
              href="/country"
            >
              Country
            </a>
          </div>
        </nav>
      </div>
      <div>
        <h1>
          Covid-19 Metrics
        </h1>
        <div
          class="sc-ftvSup kEhnWs"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
